app/models/user.rb

```diff
@@ -5,6 +5,10 @@ class User < ApplicationRecord
   has_many :podcasts
   has_many :following_podcasts, through: :followings, source: :podcast
   has_many :applications
+  
+  def pending_podcasts
```

Comment:
So for this I was thinking way too into it. All podcasts are default to false so that means the admin will have to approve them so the dashboard just lists all podcasts until approval.

Reply:
Add to the podcast model and call self.pending.

Reply:
add model test